excel/Concepts/Excel-Performance/excel-tips-for-optimizing-performance-obstructions.md

```diff
@@ -116,7 +116,9 @@ Dynamic ranges have the following advantages and disadvantages:
 
 ## Improve lookup calculation time
 
-Lookups are frequently significant calculation obstructions. Fortunately, there are many ways of improving lookup calculation time. If you use the exact match option, the calculation time for the function is proportional to the number of cells scanned before a match is found. For lookups over large ranges, this time can be significant.
+In Excel 16.0.9126.2xxx onwards VLOOKUP, HLOOKUP and MATCH for exact match on unsorted data is much faster than ever before when looking up multiple columns (or rows with HLOOKUP) from the same table range.
```

Comment:
Please change "In Excel 16.0.9126.2xxx onwards" to "In O365 version 1809 onwards, Excel's"

Reply:
@PrashSMSFT  Is this the change you want in both topics?

In Office 365 version 1809 and later, Excel's VLOOKUP, HLOOKUP, and MATCH for exact match...